src/ChineseNumbers/ChineseNumbersTest.cpp

```diff
@@ -0,0 +1,36 @@
+#include "ChineseNumbers.h"
+#include "gtest/gtest.h"
+
+namespace ChineseNumbers {
+
+TEST(ChineseNumberTest, Test1) {
+  std::string output =
+      ChineseNumbers::Generate("0001", "0", ChineseNumberCase::LOWERCASE);
```

Comment:
If you remove the nested `ChineseNumbers`, this can just be `Generate`.

Reply:
Fixed in 92e46601d055fb57699128d5d2cb8634a1fef339